OpenGov-OpenData/ckanext-pdfview
# [OD-1662] Add ckan 2.9 support

## [TICKET](https://opengovinc.atlassian.net/browse/OD-1662)

## Description

This PR adds ckan 2.9 support to the pdf resource preview.

The following changes are made:
- Import static files using either snippets for either asset (ckan 2.9 or higher) or resource (lower than ckan 2.9)
- Update icon
- Removes "resource_view.html" template as it is the same as the default template with minor differences in html class
  - This template was also removed in upstream fork.

## Test Procedure

- Install this plugin
- Enable `resource_proxy` and `pdf_view` plugin in ckan ini file
```
ckan.plugins = stats resource_proxy pdf_view
```
- Create a resource with a link to a PDF file and using the pdf format
- Create a pdf view for the PDF resource and confirm that it works in ckan 2.7 and ckan 2.9

## Approval Criteria


## Submitter Checklist
